### PR TITLE
Added handling for if Decimal is NaN

### PR DESCRIPTION
**Pull request by umbernhard**
Occasionally, in our sample size probability computation, some of the internal variables can become NaN (due to the way the stats math works, if the numbers are too big/small). This makes sure that we gracefully handle that case.

### CODE REVIEW

File: server/tests/audit_math/test_bravo.py [hunk 1]
```diff
@@ -395,6 +395,10 @@ def test_compute_risk(contests):
             ("Biden", "Overvote"): 0.0,
             ("Biden", "Undervote/Blank"): 0.0,
         },
+        "test_caroline": {
```
Comment (arsalansufi) [inline server/tests/audit_math/test_bravo.py:398]: As much as I love calling this the Caro test 😆, a name that describes the situation might be more helpful down the road

Comment (arsalansufi) [inline server/tests/audit_math/test_bravo.py:398]: As much as I love calling this the Caro test 😆, a name that describes the situation might be more helpful down the road

Comment (umbernhard) [reply server/tests/audit_math/test_bravo.py:398]: :sweat_smile: fair point!

**Review by arsalansufi — APPROVED** [review]
LGTM!

**Pull request merged**